### DISCUSSION TITLE
windows版如何修改同步时间

### DISCUSSION THREAD

**Discussion by 1158681866** [jeessy2/ddns-go#790] [Q&A]
如题windows版如何修改同步时间

**Comment by WaterLemons2k** [comment]
使用参数 `-f` 更改同步间隔时间(秒)。

示例：

```
ddns-go -f 60
```

**Reply by AngeCoo** [reply]
求大佬能在详细一点解释么，是先cd进入ddns-go的安装目录吧，然后用  ddns-go -f 60 命令启动ddns-go并把同步时间改为60秒么，但是我用教程把ddns-go安装为服务了跟随windows系统启动，我想修改同步时间需要怎么做

**Reply by WaterLemons2k** [reply]
```
ddns-go -s install -f 60
```

**Reply by AngeCoo** [reply]
感谢大佬的回复 ddns-go -s install -f 60 这个命令是执行一次就修改了启动没配么，后续跟随服务启动就是按60秒同步么
